pallets/randomness/src/lib.rs

```diff
@@ -325,7 +325,14 @@ pub mod pallet {
 		/// Uses the vrf output of previous block to generate a random seed. The provided `subject`
 		/// must have the property to uniquely generate different randomness given the same vrf
 		/// output (e.g. relay block number).
-		/// Note: This needs to be updated when asynchronous backing is in effect.
+		///
+		/// In our case the `subject` is provided via Nimbus and consists of three parts:
+		///       1. Constant string *b"filter" - to identify author-slot-filter pallet
+		///       2. First 2 bytes of index.to_le_bytes() when selecting the ith eligible author
+		///       3. First 4 bytes of slot_number.to_be_bytes()
+		///
+		/// Note: This needs to be updated when asynchronous backing is in effect,
```

Comment:
`s/Note/TODO/` ?

Reply:
:D I'm agnostic here. Will create a tracking ticket for it anyhow. Not a huge fan of TODOs if they're not currently actionable. 
Didn't we wanna remove TODOs from our codebase though?